### ISSUE TITLE
RFE: Gantt chart for scheduled jobs

### ISSUE THREAD

**Issue by sandrich** [ansible/awx#4449]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
The ability to have some sort of graphical view on scheduled jobs per day/week/month/year. One example would be a Gantt chart.

The idea behind is to have a better overview when jobs are scheduled and to find busy or non busy spots. If the job has run before you could use the average execution time for the job length estimation in the diagram.

**Comment by wenottingham** [comment]
If we've got D3 lying around, something like the github activity graph may be useful?

**Comment by sandrich** [comment]
> If we've got D3 lying around, something like the github activity graph may be useful?

@wenottingham if you are talking about the heatmap then that I find almost better than a gantt chart. It would give an easy way of finding sweet spots. As long as you can drill further in to it until you get a day view.

Also it should consider scheduled workflows and its jobs.

**Comment by tentator** [comment]
@wenottingham I think this feature would really be something! +1

**Comment by mschibli** [comment]
@wenottingham , Yeah, for me, this feature would be really a great idea! hope this will be implemented. +1
